i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.mdx

```diff
@@ -760,7 +795,18 @@ Release date: `April 26, 2023`
 
 ## Centreon IT Edition Extensions
 
-### 23.104.2
+### 23.04.3
+
+Release date: `March 12, 2024`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- When the license is not valid, expired, renamed, or moved, users now get an error page instead of multiple error messages.
```

Comment:
I think it's not the correct fix.

it's concerns : 
The IT Edition module does not generate a license error snackbar, and the custom page is not displayed if the license is invalid.